tests/loaders_test.py

```diff
@@ -347,7 +347,7 @@ def test_raises_on_missing_config():
                 mock_rule_open.return_value = test_rule_copy
                 with mock.patch('os.walk') as mock_walk:
                     mock_walk.return_value = [('', [], ['testrule.yaml'])]
-                    with pytest.raises(EAException, message='key %s should be required' % key):
```

Comment:
See [here](https://docs.pytest.org/en/6.2.x/deprecations.html#message-parameter-of-pytest-raises) for more information on why the `message` has been removed.